README.md

```diff
@@ -154,7 +154,7 @@ You can also install the PySide2 dependency with `python -m pip install .[view]`
 
 You may need to escape the `[` depending on your shell, `\[view\]` .
 
-Currently the code base is written against python 3.7, 3.8, 3.9, 3.10 and 3.11,
+Currently the code base is written against python 3.9, 3.10 and 3.11,
```

Comment:
I feel like we should say that we support 3.9 through 3.12, because we already ship wheels for 3.12.

Reply:
Gotcha.  I held off on that because if isn't in the VFX platform but you're right, we should just put it out there.